src/borg/repository.py

```diff
@@ -22,6 +22,7 @@
 from .helpers import ProgressIndicatorPercent
```

Comment:
^ "all Repository-side logging goes through the pipe" actually not the case for borg.debug. because it's disabled by default,... borg serve may just turn it on always, the client will filter it.